src/modules/initcpiocfg/main.py

```diff
@@ -115,7 +115,7 @@ def modify_mkinitcpio_conf(partitions, root_mount_point):
     btrfs = ""
     lvm2 = ""
     hooks = ["base", "udev", "autodetect", "modconf", "block", "keyboard",
-             "keymap"]
+             "keymap","consolefont"]
```

Comment:
Does the consolefont hook **always** exist? Or is this something your distro has (maybe all Arches?) but others do not? It would need a detection mechanism (hooks in general, might be nice to check that they exist before enabling them).